src/scss/_code.scss

```
@@ -56,6 +57,23 @@ pre {
     tab-size: 4;
 }
 
+div.highlight.line-numbers {
+    pre.chroma {
+        code {
+            counter-reset: line-numbering;
+            span.line::before {
+                color: theme("colors.gray.600");
+                content: counter(line-numbering);
+                counter-increment: line-numbering;
+                display: inline-block;
+                margin-right: 10px;
+                text-align: right;
+                width: 1.5em;
```

Minor suggestion to use Tailwind helpers for these:

```suggestion
                @apply text-gray-600 text-right inline-block mr-3 w-6; 
                content: counter(line-numbering);
                counter-increment: line-numbering;
```

Will take a look locally and see if this looks right, though.

---

Yep, looks good locally. 👍

---

:+1: thanks! I don't know Tailwind 😄